dht_net.go

```diff
@@ -107,6 +109,8 @@ func (dht *IpfsDHT) handleNewMessage(s network.Stream) bool {
 				ctx,
 				[]tag.Mutator{tag.Upsert(metrics.KeyMessageType, "UNKNOWN")},
 				metrics.ReceivedMessageErrors.M(1),
+				metrics.ReceivedMessages.M(1),
+				metrics.ReceivedBytes.M(int64(req.Size())),
```

Comment:
Should be `len(msgbytes)` but we need to record that before we release the message.

Reply:
it could *always* be `len(msgbytes)`, right ? Does it ever make sense to use `req.Size()` instead?

Reply:
I did that, let me know if that doesn't make sense.

Reply:
> Does it ever make sense to use

For requests, no.